src/app/app-routing.module.ts

```diff
@@ -1,7 +1,13 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { FeedComponent } from './components/feed/feed.component';
 
-const routes: Routes = [];
+const base = 'api/v1/';
```

Comment:
router should NOT point to `api/v1`. Server api lives there. Routes should be without this.